docs/reference/triggers/kafka.md

```diff
@@ -19,4 +20,7 @@ triggers:
     attributes:
       topic: "my.topic"
       partitions: [0, 5, 10]
+      SASL:
```

Comment:
Convention - camel case

Reply:
This is an acronym, like URL and HTTP. See [here](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).

Reply:
Acronyms that start words are always all lowercase in nuclio (same as in k8s).
Also you used `User` and `Password` - should be `user` and `password`